rxmongo/src/main/scala/akka/contrib/persistence/mongodb/RxMongoJournaller.scala

```diff
@@ -47,7 +49,9 @@ class RxMongoJournaller(driver: RxMongoDriver) extends MongoPersistenceJournalli
   private[mongodb] override def batchAppend(writes: ISeq[AtomicWrite])(implicit ec: ExecutionContext):Future[ISeq[Try[Unit]]] = {
     val batch = writes.toStream.map(aw => Try(driver.serializeJournal(Atom[BSONDocument](aw, driver.useLegacySerialization))))
     Future.sequence(batch.map {
-      case Success(document:BSONDocument) => journal.insert(document, writeConcern).map(writeResultToUnit)
```

Comment:
Same as Casbah - can this be changed to return the results of the journal insert rather than the realtime insert?